lib/hanami/application.rb

```diff
@@ -174,12 +175,16 @@ def routes(&block)
       MODULE_DELIMITER = "::"
       private_constant :MODULE_DELIMITER
 
-      def application_namespace
+      def namespace
```

Comment:
Renamed this (and added `namespace_path` below) to make it so that the application and slices can offer a consistent interface as much as possible. This is what enables the "hidden" feature of being able to create application-level views using `Hanami::View[Hanami.application]`.

This is _not_ something we will offer via generators or anything, but given that we were already 95% of the way there, I felt it was worth bringing these in line at the same time.